### PR TITLE
update pct_fixed_each_ts.ipynb to match template

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies

 #183 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

Added "Save to Ceph" and "Save to Disk" sections, prose, and converted series data into dataframes in order to save as parquets.

### CODE REVIEW

**Comment by isabelizimm** [comment]
No merging yet, need to prep for automation :)

File: notebooks/data-sources/TestGrid/metrics/pct_fixed_each_ts.ipynb [hunk 1]
```diff
@@ -2,20 +2,24 @@
  "cells": [
```
Comment (MichaelClifford) [inline notebooks/data-sources/TestGrid/metrics/pct_fixed_each_ts.ipynb:2]: <p>See the import cell example from <code>correlated_failures.ipynb</code>  below.   If we import from the metrics template we don't need to include the sys.path  stuff and the "#noqa" flags can be removed.  Also, you need t add the <code>load_dotenv(find_dotenv())</code>  too.  </p><pre class="ql-syntax" spellcheck="false">
# Import libraries
import gzip
import json
import os
import numpy as np
import pandas as pd
import datetime

from ipynb.fs.defs.metric_template import decode_run_length
from ipynb.fs.defs.metric_template import CephCommunication
from ipynb.fs.defs.metric_template import save_to_disk, read_from_disk
from dotenv import load_dotenv, find_dotenv

load_dotenv(find_dotenv())
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MichaelClifford) [inline notebooks/data-sources/TestGrid/metrics/pct_fixed_each_ts.ipynb:2]: <p>This looks like its also saving something to disk. This should be handled in the cells below. </p><p> </p><p>Also, here you are saving something called, <code>pct_fixed_per_ts</code>  and in the next cell you are saving <code>pct_fixed_per_ts_df</code>  this section should save all outputs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by MichaelClifford** [comment]
Is this right @oindrillac @isabelizimm? that the `converted time to test nb to automation metric template` commit should be part of this PR as well?

**Comment by oindrillac** [comment]
> Is this right @oindrillac @isabelizimm? that the `converted time to test nb to automation metric template` commit should be part of this PR as well?

@MichaelClifford that doesnt seem right..

**Comment by isabelizimm** [comment]
> Is this right @oindrillac @isabelizimm? that the `converted time to test nb to automation metric template` commit should be part of this PR as well?

@MichaelClifford not supposed to be in there...but I have no clue how it managed to join 😅

**Pull request merged**